src/pyasm/command/file_upload.py

```diff
@@ -175,11 +175,26 @@ def _dump_file_to_temp(my):
             return
 
         System().makedirs(dirname)
+   
+        # Get temporary file path to read from
+        # Linux uses mkstemp, while Windows uses TemporaryFile
+        if os.name == 'nt':
+            data = my.field_storage.file
```

Comment:
Move check for base64 upload to upload_server_page, only check on create action OR html5 upload action, 
pass decode paramter to file_upload
Store base64 indicator file in from path